src/mavedb/scripts/map_to_uniprot_id_from_mapped_metadata.py

```diff
@@ -0,0 +1,97 @@
+import click
+import logging
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+from mavedb.scripts.environment import with_database_session
+from mavedb.models.score_set import ScoreSet
+from mavedb.lib.uniprot.id_mapping import UniProtIDMappingAPI
+from mavedb.lib.uniprot.utils import infer_db_name_from_sequence_accession
+from mavedb.lib.mapping import extract_ids_from_post_mapped_metadata
+
+VALID_UNIPROT_DBS = [
+    "UniProtKB",
+    "UniProtKB_AC-ID",
+    "UniProtKB-Swiss-Prot",
+    "UniParc",
+    "UniRef50",
+    "UniRef90",
+    "UniRef100",
+]
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@with_database_session
+@click.option("--score-set-urn", type=str, default=None, help="Score set URN to process. If not provided, process all.")
+@click.option("--polling-interval", type=int, default=3, help="Polling interval in seconds for checking job status.")
```

Comment:
No change required, but I was finding the default interval to be too short here, and reaching the max number of retries before the job was finished. Results may vary, but I had better success when setting this to 40.